### DISCUSSION TITLE
Bugs and issues

### DISCUSSION THREAD

**Discussion by codeabuse** [Anarios/return-youtube-dislike#918] [General]
First of all, hello and thank you for your add-on!

Here's my report on v3.0.0.9 for Firefox.
- 'Like' counter on thumbs-up button is showing dislikes count,
- 'Dislike' button shows no counter,
- The only valid info is shown when hovering likes/dislikes bar.
Everything is shown on the screenshot:
![image](https://github.com/Anarios/return-youtube-dislike/assets/47417014/09c02274-9cbd-4de7-891e-3b09fa05d856)

**Comment by iAmNikola** [comment]
This is just a pure guess, but it might be because of the download button add-on. It might be injecting values into the first and second element of the yt like HTML element (download button and like button in your case).

I recommend trying to disable the download add on and seeing if that fixes the issue 🙂

**Reply by iAmNikola** [reply]
I just saw your issue. My guess was right, but I was beat to the punch lol
